msibi/pair.py

```diff
@@ -92,16 +92,14 @@ def compute_current_rdf(self, state, r_range, n_bins, smooth=True,
         g_r_all = None
         first_frame = 0
         max_frames = int(max_frames)
-        for last_frame in range(max_frames,
-                                state.traj.n_frames + max_frames,
-                                max_frames):
-            r, g_r = md.compute_rdf(state.traj[first_frame:last_frame],
-                                    pairs, r_range=r_range / 10, n_bins=n_bins)
+        for last_frame in range(max_frames,state.traj.n_frames + max_frames, max_frames):
+            r, g_r = md.compute_rdf(state.traj[first_frame:last_frame], pairs, r_range= r_range, n_bins=n_bins)
```

Comment:
Most of this wasn't intended changes. This github interface is confusing me but `r_range / 10` was probably just a conversion from A to nm.